lib/smart_answer/calculators/minimum_wage_calculator.rb

```diff
@@ -208,5 +208,13 @@ def charged_accomodation_adjustment(charge, number_of_nights)
     def self.historical_minimum_wage_data
       @historical_minimum_wage_data ||= YAML.load_file(Rails.root.join("lib/data/minimum_wage_data.yml"))[:minimum_wage_data]
     end
+
+    def date_from_environment_variable
```

Comment:
@leenagupte Since someone else is going to be taking over Smart Answers next week (and given it's 4:30PM on Friday now), I think that should maybe be a new card. What do you reckon?

I can walk through the required changes with the next person during the handover meeting (10am Monday).